src/Apps/Consign/Routes/ConsignmentInquiry/Components/ConsignmentInquiryFormAbandonEdit.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button, Flex, ModalDialog, Spacer, Text } from "@artsy/palette"
+import { useRouter } from "System/Router/useRouter"
+
+export const ConsignmentInquiryFormAbandonEditModal: React.FC<{
+  show: boolean
+  onClose: () => void
+}> = ({ show, onClose }) => {
+  const { router } = useRouter()
+
+  if (!show) {
+    return null
+  }
```

Comment:
**suggestion:** if possible, I would prefer moving this check to the parent component